lib/osf-components/addon/components/registries/review-metadata-renderer/label-display/template.hbs

```diff
@@ -0,0 +1,20 @@
+{{assert 'Registries::ReviewMetadataRenderer::LabelDisplay requires a draftRegistration' @draftRegistration}}
+{{assert 'Registries::ReviewMetadataRenderer::LabelDisplay requires a field' @field}}
+{{assert 'Registries::ReviewMetadataRenderer::LabelDisplay requires a changeset' @changeset}}
+<h3>
```

Comment:
On the review page, labels for other fields are `<p>`. Is there any reason why we want to make the labels for metadata fields `<h3>`?